tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py

```diff
@@ -0,0 +1,202 @@
+        return False
+
+    return True
+
+
+def check_total_space_after_expansion(total_space_b4_expansion):
+    # The newly added capacity takes into effect at the storage level
+    ct_pod = pod_helpers.get_ceph_tools_pod()
+    output = ct_pod.exec_ceph_cmd(ceph_cmd='ceph osd df')
+    total_space_after_expansion = int(output.get('summary').get('total_kb'))
+    osd_size = int(output.get('nodes')[0].get('kb'))
+    expanded_space = osd_size * 3  # 3 OSDS are added of size = 'osd_size'
+    logging.info(f"expanded_space == {expanded_space} ")
+    logging.info(f"space output == {output} ")
+    logging.info(f"osd size == {osd_size} ")
+    logging.info(f"total_space_after_expansion == {total_space_after_expansion} ")
+    expected_total_space_after_expansion = total_space_b4_expansion + expanded_space
+    logging.info(f"expected_total_space_after_expansion == {expected_total_space_after_expansion} ")
+    if not total_space_after_expansion == expected_total_space_after_expansion:
+        return False
+
+    return True
+
+
+def check_osd_tree():
+    # 'ceph osd tree' should show the new osds under right nodes/hosts
+    #   Verification is different for 3 AZ and 1 AZ configs
+    osd_pods = pod_helpers.get_osd_pods()
+    ct_pod = pod_helpers.get_ceph_tools_pod()
+    tree_output = ct_pod.exec_ceph_cmd(ceph_cmd='ceph osd tree')
+    logging.info(f"### OSD tree output = {tree_output}")
+    if config.ENV_DATA['platform'] == 'vsphere':
+        return temp_helper_file.check_osd_tree_1az_vmware(tree_output, len(osd_pods))
+
+    aws_number_of_zones = 3
+    if config.ENV_DATA['platform'] == 'AWS':
+        # parse the osd tree. if it contains a node 'rack' then it's a AWS_1AZ cluster. Else, 3 AWS_3AZ cluster
+        for i in range(len(tree_output['nodes'])):
+            if tree_output['nodes'][i]['name'] in "rack":
+                aws_number_of_zones = 1
+        if aws_number_of_zones == 1:
+            return temp_helper_file.check_osd_tree_1az_aws(tree_output, len(osd_pods))
+        else:
+            return temp_helper_file.check_osd_tree_3az_aws(tree_output, len(osd_pods))
+
+
+@pytest.mark.parametrize(
+    argnames=["node_type", "num_of_nodes"],
+    argvalues=[
+        pytest.param(
+            *['worker', 1],
+            marks=pytest.mark.polarion_id("OCS-1311")
+        ),
+     ]
+)
+@pytest.mark.parametrize(
+    "workload_storageutilization_rbd",
+    [(0.02, False, 120)],
+    indirect=["workload_storageutilization_rbd"]
+)
+# @pytest.mark.parametrize(
+#     "workload_storageutilization_cephfs",
+#     [(0.05, False, 120)],
+#     indirect=["workload_storageutilization_cephfs"]
+# )
+@ignore_leftovers
+@tier4
+@polarion_id('OCS-1313')
+class TestAddCapacityNodeRestart(ManageTest):
+    """
+    Test add capacity when one of the nodes got restart
+    in the middle of the process. Don't forget to include the flag '--cluster-name' when running the test
+    """
+    num_of_pvcs = 4
+    pvc_size = 100
+
```

Comment:
Looks like this test is not running any background IOs. As per test case OCS-1313, "Make sure this test has the configuratio CFG#1 in https://docs.google.com/spreadsheets/d/1yMzw6SzWwcrB1nM5atEVVNc9FbpRV9cOEUDPlteMQoo/edit#gid=978210857&range=C3 and Backgroud operatios as mentioned in https://docs.google.com/spreadsheets/d/1yMzw6SzWwcrB1nM5atEVVNc9FbpRV9cOEUDPlteMQoo/edit#gid=978210857&range=B17". Please try to follow the entry criteria mentioned there.

Reply:
As I wrote in my comment here: https://github.com/red-hat-storage/ocs-ci/pull/1996/files#diff-b451e3caa51bd819db2305fc5466e846R44 these changes will be replaced when the branch 'wip-add-capacity-e_e' will be merged into master.